src/stk/molecular/molecules/molecule/utilities/updaters/turbomole.py

```diff
@@ -25,16 +26,39 @@ def _with_structure_from_turbomole(self, path):
         number of atoms in the molecule or if atom elements in the
         file do not agree with the atom elements in the molecule.
 
+    :class:`NotImplementedError`
+        If the the Turbomole file has coordinates defined as fractional
+        based on a unit cell.
+
     """
 
     bohr_to_ang = 0.5291772105638411
 
     with open(path, 'r') as f:
-        _, *content, __ = f.readlines()
+        content = f.readlines()
+
+    for i, cont in enumerate(content):
```

Comment:
Also not a huge for of `i`, I know its a common convention, but `line_number` I think is more clear, especailly because later you define `j` too at which point I think the code becomes ... coded.

Reply:
Done.